src/router.js

```diff
@@ -0,0 +1,28 @@
+export default class Router {
+  constructor(routes, element) {
+    this.routes = routes;
+    this.rootElement = element;
+    this.init();
+  }
+
+  init() {
+    window.addEventListener('hashchange', (url) => { this.render(url); });
+    window.location.hash = 'login';
+  }
```

Comment:
Also here we should add something like "this.render(currentUrl)". As we discussed before this need to proceed current url when index.js was loaded for a first time.